packages/mdc-slider/mdc-slider.scss

```diff
@@ -0,0 +1,226 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/theme/mixins";
+@import "@material/rtl/mixins";
+@import "./variables";
+
+$mdc-slider-main-grey: #bdbdbd;
+$mdc-slider-dark-theme-grey: #5c5c5c;
+
+// postcss-bem-linter: define slider
+
+@keyframes mdc-slider-emphasize {
+  0% {
+    animation-timing-function: ease-out;
+  }
+
+  50% {
+    animation-timing-function: ease-in;
+    transform: scale(.85);
+  }
+
+  100% {
+    transform: scale(.571);
+  }
+}
+
+.mdc-slider {
+  position: relative;
+  width: 100%;
+  height: 48px;
+  cursor: pointer;
+  touch-action: pan-x;
+  -webkit-tap-highlight-color: rgba(black, 0);
+
+  &:focus {
+    outline: none;
+  }
+
+  &__track-container {
+    position: absolute;
+    top: 50%;
+    width: 100%;
+    height: 2px;
+    background-color: $mdc-slider-main-grey;
+    overflow: hidden;
+
+    @include mdc-theme-dark(".mdc-slider") {
+      background-color: $mdc-slider-dark-theme-grey;
+    }
+  }
+
+  &__track {
+    @include mdc-theme-prop(background-color, primary);
+
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    transform-origin: left top;
+    will-change: transform;
+
+    @include mdc-rtl(".mdc-slider") {
+      transform-origin: right top;
+    }
+  }
+
+  &__thumb-container {
+    position: absolute;
+    top: 15px;
+    left: 0;
+    width: 21px;
+    // Ensure that touching anywhere within the Y-coordinate space of thumb
+    // is considered "clicking on the thumb".
+    height: 100%;
+    will-change: transform;
+  }
+
+  &__thumb {
+    @include mdc-theme-prop(fill, primary);
+    @include mdc-theme-prop(stroke, primary);
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    transform: scale(.571);
+    transition: transform 100ms ease-out, fill 100ms ease-out, stroke 100ms ease-out;
+    stroke-width: 3.5;
+  }
+
+  &__focus-ring {
+    @include mdc-theme-prop(background-color, primary);
+
+    width: 21px;
+    height: 21px;
+    transition: transform 266.67ms ease-out, opacity 266.67ms ease-out, background-color 266.67ms ease-out;
+    border-radius: 50%;
+    opacity: 0;
+  }
+}
+
+.mdc-slider--active {
+  &:not(.mdc-slider--disabled) .mdc-slider__thumb {
```

Comment:
yeah dunno why I did that...removing `:not(.mdc-slider--disabled)` since the slider can't ever be active if also disabled